cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/common/MeasureEvaluator.java

```diff
@@ -597,7 +596,7 @@ protected void evaluateCohort(
         if (initialPopulation == null) {
             throw new NullPointerException("`" + INITIALPOPULATION.getDisplay()
                     + "` is a required Population Definition for Measure Scoring Type: "
-                    + measureDef.scoring().get(groupDef).toCode());
```

Comment:
The measureDef parameter is no longer used at all in this method.  Consider removing it or add a comment about why it's there.